src/components/main/compose/msg/mod.rs

```diff
@@ -235,6 +236,58 @@ pub fn Msg<'a>(cx: Scope<'a, Props>) -> Element<'a> {
                                 onpressed: move |_| popout.set(true),
                                 text: String::from("Reply"),
                                 icon: Shape::ArrowUturnLeft,
+                            },
+                            ContextItem {
```

Comment:
it looks like this is deleting the conversation? I don't think we want to do that. I think we want to be able to remove a conversation from the sidebar, but then get that conversation back by clicking on the chat icon from the Friends list.

Reply:
Resolved by the latest commit